### PR TITLE
feature_importances quick method

### PR DESCRIPTION
**Pull request by rebeccabilbro**
Refactors the `feature_importances` quick method to the new quick method signature and verifies that it works correctly (towards #960)

# TODOs specific to this PR
- [x] check quick method docstring and signature
- [x] add the show kwarg and implementation
- [x] verify the quick method tests
- [x] run locally to ensure it works as expected
- [x] update the documentation
- [ ] run tests in CI and watch something fail 😢 

# General YB PR checklist
- [x] _Is the commit message formatted correctly?_
- [ ] _Have you noted the new functionality/bugfix in the release notes of the next release?_
- [ ] _Included a sample plot to visually illustrate your changes?_
- [x] _Do all of your functions and methods have docstrings?_
- [x] _Have you added/updated unit tests where appropriate?_
- [ ] _Have you updated the baseline images if necessary?_
- [x] _Have you run the unit tests using `pytest`?_
- [x] _Is your code style correct (are you using PEP8, pyflakes)?_
- [x] _Have you documented your new feature/functionality in the docs?_
- [x] _Have you built the docs using `make html`?_

### CODE REVIEW

File: docs/api/model_selection/importances.rst [hunk 1]
```diff
@@ -16,8 +16,16 @@ other. In scikit-learn, Decision Tree models and ensembles of trees such as
 Random Forest, Gradient Boosting, and Ada Boost provide a
 ``feature_importances_`` attribute when fitted. The Yellowbrick
 ``FeatureImportances`` visualizer utilizes this attribute to rank and plot
-relative importances. Let's start with an example; first load a
```
Comment (bbengfort) [inline docs/api/model_selection/importances.rst]: Good edit!

File: docs/api/model_selection/importances.rst [hunk 2]
```diff
@@ -126,11 +134,28 @@ In both cases, because the coefficient may be negative (indicating a strong nega
 
 This method may also be used for instances; but generally there are very many instances relative to the number models being compared. Instead a heatmap grid is a better choice to inspect the influence of features on individual instances. Here the grid is constructed such that the x-axis represents individual features, and the y-axis represents individual instances. The color of each cell (an instance, feature pair) represents the magnitude of the product of the instance value with the feature's coefficient for a single model. Visual inspection of this diagnostic may reveal a set of instances for which one feature is more predictive than another; or other types of regions of information in the model itself.
 
+The same functionality above can be achieved with the associated quick method `feature_importances`. This method will build the ``FeatureImportances`` object with the associated arguments, fit it, then (optionally) immediately show it.
```
Comment (bbengfort) [inline docs/api/model_selection/importances.rst]: Not sure how important this is, the other documentation has been adding a header here, e.g.

```
Quick Method
------------
```

This has the effect of creating a link in the table of contents for the quick method when you're on this page. Do you want to standardize on this or does it matter?

Comment (rebeccabilbro) [reply docs/api/model_selection/importances.rst]: Ah thank you, yes &mdash; definitely standardization seems like the right way to go! I was going off the [quick methods example](http://www.scikit-yb.org/en/develop/api/cluster/silhouette.html) from the updated `SilhouetteVisualizer` docs. I'll update my docs here (and in my other QM PRs) and in Silhouette for consistency's sake.

Comment (bbengfort) [reply docs/api/model_selection/importances.rst]: Sounds good, feel free to merge when you're ready!

**Pull request merged**